testing/widgets/common/EditorAppearanceVariants.test.js

```diff
@@ -0,0 +1,46 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import asyncForEach from '../../../utils/visual-tests/helpers/async-for-each';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+fixture('Common.EditorAppearanceVariants')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
+    await t.resizeWindow(900, 600);
+  });
+
+runManualTest('Common', 'EditorAppearanceVariants', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
```

Comment:
Let's remove `jquery` approach from testing because we reload the page after change of `stylingMode` to apply it to all editors from global config. It can cause test fails and to do it will better than rewrite the scenario or to have unstable or red test. 
You may add comment why we decide not to test this.